content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -2520,53 +2519,287 @@ Optional attributes can be added to the root element `kanban` to customize the v
 Components
 ----------
 
-Kanban views accept the following children elements: :ref:`field
+Kanban views accept the following children elements: :ref:`templates
+<reference/view_architectures/kanban/templates>`, :ref:`field
 <reference/view_architectures/kanban/field>`, :ref:`header
 <reference/view_architectures/kanban/header>`, :ref:`progressbar
-<reference/view_architectures/kanban/progressbar>`, and :ref:`templates
-<reference/view_architectures/kanban/templates>`.
+<reference/view_architectures/kanban/progressbar>`.
 
-Placeholders are denoted in all caps.
+.. _reference/view_architectures/kanban/templates:
 
-.. _reference/view_architectures/kanban/field:
+`templates`: define cards structure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-`field`: display field values
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The `templates` element is used to define the :ref:`QWeb templates <reference/qweb>` that structure
+the kanban cards.
```

Comment:
just to make sure, this is qweb, but not owl right? we should probably mention that we cannot do a `t-on-click` for example?